apiary.apib

```diff
@@ -689,10 +689,17 @@ Import a tar file into ipfs
 
 + Response 200 (text/plain)
 
-## cat [GET /tar/cat]
```

Comment:
Good question. Rebase mess up. Removed cat in #1 by accident; added it back in properly now. Thanks for the catch. Clearly my bad.